### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by bajajcodes**
## Cart Page
- Cart Page is made up of four components Header, Aside, Main and Footer.
- Header component is used for navigation to profile, wishlist and cart page, and for login or signup.
- Aside component is used for summarising total amount for products in card, it uses Card component with CSS overriding.
- Main component consist of page title element and cards container which contains products using Card component, and provide UI features to remove from cart, increase or decrease quantity, add or remove from wish-list.
- Footer component is used for giving some extra information.

## What files to check
- [ ] components/card/card.js
- [ ] pages/cart/

## Suggestions for review
- UI interactivity
- Responsiveness
- File naming consistency
- Code readability and consistency in writing code in same way. 

## Preview
https://user-images.githubusercontent.com/29247011/159242561-ac9acd64-497b-4563-bddc-34363f80c2fa.mp4

### CODE REVIEW

File: src/pages/cart/aside/aside.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+</div>  
+</div>
+
+<div className="card card-with-no-border card-w-100 card-w-100">
+<div className="card-footer card-footer-dir-col">
+<div className="cart-items-category">
+            <span>Price (2 items)</span>
+            <span>₹4999</span>
+  </div>
+<div className="cart-items-category">
```
Comment (anuj-kumary) [inline src/pages/cart/aside/aside.js]: Here Do some code formatting.

**Review by anuj-kumary — APPROVED** [review]
- Good naming conventions
- Clean code
- folder structure is also good

File: src/pages/cart/aside/aside.js (hunk 1 above)
Comment (bajajcodes) [reply src/pages/cart/aside/aside.js]: Changes made.

File: src/components/card/card.js [hunk 2]
```diff
@@ -1,49 +1,66 @@
...
+            onClick={() => addToCartHandler()}
+          >
+            <span className="material-icons-outlined">shopping_cart</span>
+            {inCart ? "Remove fom cart" : "Add to Cart"}
+          </button>
+          <button
+            className={`btn btn-outline-secondary ${
+              inWishlist ? "primrary-background" : ""
+            }`}
+            onClick={() => addToWishlistHandler()}
```
Comment (mimansha-swarup) [inline src/components/card/card.js:56]: you can write it as onClick={ addToWishlistHandler}

File: src/components/footer/footer.styles.css [hunk 3]
```diff
@@ -1,18 +1,19 @@
 .footer {
-    border-top: var(--size-1) solid;
-    display: flex;
-    padding: var(--size-9);
-    background-color: var(--white);
-  }
+  border-top: var(--size-1) solid;
+  display: flex;
+  padding: var(--size-9);
```
Comment (mimansha-swarup) [inline src/components/footer/footer.styles.css:4]: I appreciate how you have made variables for all the stuff

**Review by mimansha-swarup — APPROVED** [review]
code is clean

**Review by zoyatanweer — APPROVED** [review]
code looks good and clean.

File: src/components/card/card.js (hunk 2 above)
Comment (bajajcodes) [reply src/components/card/card.js:56]: I will implement it.

**Pull request merged**